src/js/index.js

```diff
@@ -243,6 +247,11 @@ export const hpe = deepFreeze({
     },
     extend: css`
       ${props => !props.plain && 'font-weight: bold;'}
+      ${({ primary, theme }) =>
+        primary &&
+        `color: ${
+          theme.global.colors['text-strong'][theme.dark ? 'dark' : 'light']
+        };`}
```

Comment:
It would make more sense to use `button.primary.extend` https://v2.grommet.io/button#button.primary.extend instead of the general  `button.extend`

Reply:
FYI, realized that the designs call for 'text-strong' across all buttons, not just primary, so making that adjustment.